Tasks/Maven/maventask.ts

```diff
@@ -273,23 +273,27 @@ function processMavenOutput(data) {
     }
 }
 
-// Identifies all maven modules below the root by the presence of a pom.xml file and a /target/ directory
-function findModules(folder:string):ma.ModuleAnalysis[] {
+// Identifies maven modules below the root by the presence of a pom.xml file and a /target/ directory,
+// which is the conventional format of a Maven module.
+// There is a possibility of both false positives if the above two factors are identified in a directory
+// that is not an actual Maven module, or if the module is not currently being built.
+// The possibility of false positives should be taken into account when this method is called.
+function findCandidateModules(directory:string):ma.ModuleAnalysis[] {
```

Comment:
We did, but it should go into a different pull request since this one is already fairly complex.